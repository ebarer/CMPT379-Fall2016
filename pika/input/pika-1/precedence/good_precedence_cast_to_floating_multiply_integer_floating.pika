exec {

  print [ 15 | float ] * 3.4, _n_.
  print [ 28 | float ] * 6.8, _n_.
  print [ 4 | float ] * 2.2, _n_.
  print [ 78 | float ] * 5.0, _n_.
  print [ 26 | float ] * 4.9, _n_.
  print [ 25 | float ] * 7.7, _n_.
  print [ 39 | float ] * 5.9, _n_.
  print [ 3 | float ] * 6.1, _n_.
  print [ 24 | float ] * 2.1, _n_.
  print [ 95 | float ] * 9.1, _n_.
  print [ 89 | float ] * 2.6, _n_.
  print [ 34 | float ] * 3.2, _n_.

  print 2.8 * [ 23 | float ], _n_.
  print 2.2 * [ 54 | float ], _n_.
  print 6.2 * [ 72 | float ], _n_.
  print 8.9 * [ 59 | float ], _n_.
  print 3.3 * [ 66 | float ], _n_.
  print 7.3 * [ 41 | float ], _n_.
  print 8.3 * [ 99 | float ], _n_.
  print 8.8 * [ 65 | float ], _n_.
  print 8.7 * [ 31 | float ], _n_.
  print 9.3 * [ 6 | float ], _n_.
  print 5.9 * [ 8 | float ], _n_.
  print 6.0 * [ 38 | float ], _n_.

  print [ 99 | float ] * [ 61 | float ], _n_.
  print [ 72 | float ] * [ 78 | float ], _n_.
  print [ 44 | float ] * [ 65 | float ], _n_.
  print [ 47 | float ] * [ 2 | float ], _n_.
  print [ 1 | float ] * [ 47 | float ], _n_.
  print [ 67 | float ] * [ 61 | float ], _n_.
  print [ 66 | float ] * [ 41 | float ], _n_.
  print [ 18 | float ] * [ 55 | float ], _n_.
  print [ 30 | float ] * [ 4 | float ], _n_.
  print [ 68 | float ] * [ 27 | float ], _n_.
  print [ 36 | float ] * [ 8 | float ], _n_.
  print [ 72 | float ] * [ 47 | float ], _n_.

}

