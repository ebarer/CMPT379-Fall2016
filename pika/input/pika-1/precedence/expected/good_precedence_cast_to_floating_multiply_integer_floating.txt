51
190.4
8.8
390
127.4
192.5
230.1
18.3
50.4
864.5
231.4
108.8
64.4
118.8
446.4
525.1
217.8
299.3
821.7
572
269.7
55.8
47.2
228
6039
5616
2860
94
47
4087
2706
990
120
1836
288
3384
